recipe/meta.yaml

```diff
@@ -1,41 +1,43 @@
 {% set name = "openvino" %}
-{% set version_postfix = "2023.1.0" %}
-{% set version = "2023.1.0" %}
+{% set version_postfix = "2023.2.0" %}
+{% set version = "2023.2.0" %}
 
 package:
   name: {{ name|lower }}-split
   version: {{ version }}
 
 source:
-  - url: https://github.com/openvinotoolkit/openvino/archive/refs/tags/2023.1.0.tar.gz
-    sha256: ff88596b342440185874ddbe22874b47ad7b923f14671921af760b15c98aacd6
+  - url: https://github.com/openvinotoolkit/openvino/archive/refs/tags/{{ version }}.tar.gz
+    sha256: 419b3137a1a549fc5054edbba5b71da76cbde730e8a271769126e021477ad47b
     patches:
-    - patches/001-python-osx-arm64.patch  # [arm64]
-    - patches/002-win32-python-search.patch
-    - patches/003-lin-arm64-compute-lib.patch  # [aarch64]
-    - patches/004-gflags-samples.patch
-    - patches/005-macos-14.patch
-  - url: https://github.com/ARM-software/ComputeLibrary/archive/refs/tags/v23.02.1.tar.gz  # [aarch64 or arm64]
-    sha256: c3a443e26539f866969242e690cf0651ef629149741ee18732f954c734da6763  # [aarch64 or arm64]
+    - patches/003-lin-arm64-compute-lib.patch
+  - url: https://github.com/ARM-software/ComputeLibrary/archive/refs/tags/v23.08.tar.gz  # [aarch64 or arm64]
+    sha256: 62f514a555409d4401e5250b290cdf8cf1676e4eb775e5bd61ea6a740a8ce24f  # [aarch64 or arm64]
     folder: src/plugins/intel_cpu/thirdparty/ComputeLibrary  # [aarch64 or arm64]
-  - url: https://github.com/openvinotoolkit/oneDNN/archive/a1aa20ca8f19465dc2fd18389953ed83798b2fd3.tar.gz
-    sha256: d97ebb36cec6df7ba5473ecee38f0e49e6bda731b0414331b531dc8d1b5b227a
+  - url: https://github.com/openvinotoolkit/oneDNN/archive/2ead5d4fe5993a797d9a7a4b8b5557b96f6ec90e.tar.gz
+    sha256: 3c51d577f9e7e4cbd94ad08d267502953ec64513241dda6595b2608fafc8314c
     folder: src/plugins/intel_cpu/thirdparty/onednn
-  - url: https://github.com/openvinotoolkit/mlas/archive/c7c8a631315000f17c650af34431009d2f22129c.tar.gz
```

Comment:
We should add
`  - url: https://github.com/openvinotoolkit/mlas/archive/f6425b1394334822390fcd9da12788c9cd0d11da.tar.gz
    sha256: 707a6634d62ea5563042a67161472b4be3ffe73c9783719519abdd583b0295f4
    folder: src/plugins/intel_cpu/thirdparty/mlas`